src/workerd/api/queue.h

```diff
@@ -285,15 +305,9 @@ class QueueCustomEventImpl final: public WorkerInterface::CustomEvent, public kj
   QueueEventResult result;
 };
 
-#define EW_QUEUE_ISOLATE_TYPES \
-  api::WorkerQueue,                     \
-  api::WorkerQueue::SendOptions,        \
-  api::WorkerQueue::MessageSendRequest, \
-  api::IncomingQueueMessage,            \
-  api::QueueResponse,                   \
-  api::QueueMessage,                    \
-  api::QueueEvent,                      \
-  api::QueueController,                 \
-  api::QueueExportedHandler
+#define EW_QUEUE_ISOLATE_TYPES                                                                     \
+  api::WorkerQueue, api::WorkerQueue::SendOptions, api::WorkerQueue::SendBatchOptions,             \
+      api::WorkerQueue::MessageSendRequest, api::IncomingQueueMessage, api::QueueResponse,         \
+      api::QueueMessage, api::QueueEvent, api::QueueController, api::QueueExportedHandler
```

Comment:
Nit: generally prefer to keep this in the original one-item-per-line format as it makes reviewing additions/subtractions easier.